browser-test/src/support/admin_programs.ts

```diff
@@ -212,7 +212,7 @@ export class AdminPrograms {
     await this.page.click('#update-block-button:not([disabled])')
 
     for (const questionName of questionNames) {
-      await this.page.click(`button:text("${questionName}")`)
+      await this.page.click(`button >> text="${questionName}"`)
```

Comment:
Note to the reviewer: Now that we use the "Name" canonical question, the previous selector would have caught the "Edit Screen Name and Description" button, since the `text()` selector asserts that the element contains the text, not that it's an exact match.